cli/parse.go

```diff
@@ -109,6 +106,17 @@ func parse(env environ.Values, r io.Reader) (*run.Config, error) {
 	return cfg, nil
 }
 
+func getDriver(name string) (drivers.Driver, error) {
+	switch name {
+	case "postgres":
+		return &postgres.PG{}, nil
```

Comment:
since this and the mysql one are empty structs, we don't need pointers to them, so we can just do postgres.PG{}